xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -450,8 +449,6 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct style>& subSty
       style->BackColour =
           ConvColor(subStyle->shadowColor, subStyle->shadowOpacity); // Set the box shadow color
       style->Shadow = (10.00 / 100 * subStyle->shadowSize) * scale; // Set the box shadow size
-      // By default a box overlaps the other, then we increase a bit the line spacing
-      lineSpacing = 8.0 * scaleDefault;
```

Comment:
dont remove the scale, linespacing is not scaled

Reply:
The scale is applied only for the "Box" style, so unless we remove it, the spacing will behave very differently compared to the other styles. You don't expect the line spacing to suddenly change when you change the background style.

The default scale is 1.5, so I've chosen 12.0 as the default value for the spacing, which, together with the textual clarification, seems like a reasonable default for all scenarios.

Reply:
the scale change because depends on ass script configuration usually is set to fit the video stream resolution,
here its needed test better various use cases because i dont remember all behaviours atm